recipes/libboxes/all/conanfile.py

```diff
@@ -0,0 +1,122 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, replace_in_file, rm, rmdir
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+from conan.tools.microsoft import check_min_vs, is_msvc, is_msvc_static_runtime
```

Comment:
```suggestion
from conan.tools.microsoft import is_msvc
```

Please, keep only used imports.

Reply:
Will remove.  Thanks!